docs/dev/setup/jenkins-gitlab.rst

```diff
@@ -124,8 +124,46 @@ Starting the Artemis server should now succeed.
 GitLab
 ------
 
-Gitlab Server Setup
-~~~~~~~~~~~~~~~~~~~
+Gitlab Server Quickstart
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following steps describes how to set up the Gitlab server in a semi-automated way.
+This is ideal as a quickstart for developers. For a more detailed setup, see `Manual Gitlab Server Setup <#gitlab-server-setup>`__.
+
+1. Start the Gitlab container defined in `src/main/docker/gitlab-jenkins-mysql.yml` by running
+
+   ::
+
+        docker-compose -f src/main/docker/gitlab-jenkins-mysql.yml up --build
```

Comment:
```suggestion
        docker-compose -f src/main/docker/gitlab-jenkins-mysql.yml up --build -d
```
Detaching here would be nice, as then the same terminal can just be used after step 5 to terminate them as commented there.